pkg/service/backup/backupspec/manifest.go

```diff
@@ -219,28 +221,44 @@ func (m *ManifestContentWithIndex) ForEachIndexIter(cb func(fm FilesMeta)) error
 		return err
 	}
 
+	filter, err := ksfilter.NewFilter(keyspace)
```

Comment:
defer f.Close() (couple of lines above) doesn't log the potential error. 
I know it's not introduced in your PR, but lacking at least some info message when the file couldn't be closed makes tracing sometimes weird production bugs easier.
Appreciate if you can add it.